docs/getting_started/install_manually.md

```diff
@@ -6,99 +6,69 @@ You can install eZ Platform manually on the following operating systems:
 - [macOS](#installing-on-macos) (*development only*)
 - [Windows](#installing-on-windows) (*development only*)
 
-!!! caution "Supported systems"
+!!! note "Supported systems"
 
     Only installation on Linux is fully supported.
 
-    Installations on macOS or Windows can only be used for development.
+    Installations on macOS or Windows can only be used for development purposes.
 
 ## Available distributions
 
-eZ Platform exists in different distributions.
-You select the distribution when performing the `ezplatform:install` command, by using the correct installation type.
-It depends on the meta-repository you are using.
+eZ Platform exists in different distributions depending on the meta-repository you are using.
 
-!!! caution "Demo installation"
+!!! note "Demo installation"
 
     The Demo is intended for learning and inspiration. Do not use it as a basis for actual projects.
 
-### eZ Platform installation types
+eZ Platform installation types:
 
-| Type | Repository |
-|------|----------------|
-| `ezplatform-clean` | [ezplatform](https://github.com/ezsystems/ezplatform) |
-| `ezplatform-demo` | [ezplatform-demo](https://github.com/ezsystems/ezplatform-demo) |
+- [ezplatform](https://github.com/ezsystems/ezplatform) 
+- [ezplatform-demo](https://github.com/ezsystems/ezplatform-demo)
 
-### eZ Platform Enterprise Edition installation types
+eZ Platform Enterprise Edition installation types:
 
-| Type | Repository |
-|------|----------------|
-| `ezplatform-ee-clean` | [ezplatform-ee](https://github.com/ezsystems/ezplatform-ee) |
-| `ezplatform-ee-demo`  | [ezplatform-ee-demo](https://github.com/ezsystems/ezplatform-ee-demo) |
+- [ezplatform-ee](https://github.com/ezsystems/ezplatform-ee)
+- [ezplatform-ee-demo](https://github.com/ezsystems/ezplatform-ee-demo)
 
 ## Installing on Linux
 
-## 1. Install a LAMP Stack
-
-Depending on your Linux distribution, you may need to install part or all of the LAMP (Linux, Apache, MySQL, PHP) stack required to run eZ Platform.
-Before getting started, make sure you review the [requirements](requirements_and_system_configuration.md) page to see the systems we support and use for testing.
-
-You may use your system's package manager (yum, apt-get, etc.) to obtain a copy of Apache, MySQL, and PHP, or download the latest versions from the official websites and install manually:
-
--   [Apache](https://httpd.apache.org/download.cgi)
--   [MySQL](http://dev.mysql.com/downloads/mysql/)
--   [PHP 5.6+](http://php.net)
-
-For example, with Debian you can use `apt-get` to install `apache2`, `mysql-server`, `mysql-client`,
-and all PHP packages listed in the [requirements](requirements_and_system_configuration.md).
-You also need `git` for version control.
+### Prepare work environment
 
-!!! note "Set up Swap on Debian"
+Depending on your Linux distribution, you will need running LAMP stuck (Linux, Apache, MySQL, PHP). 
```

Comment:
How about `You will need a running LAMP stack (Linux, Apache, MySQL, PHP)`?
Because if it's "depending on Linux distribution", then you already have the L in Lamp